app/assets/javascripts/application.js

```diff
@@ -14,3 +14,26 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+$(document).on('ready page:load', function(){
```

Comment:
@bgdeutsch 
I would remove this js code and make its own file in assets/javascripts. The application.js file should only contain the the manifest of JS files you want to include. Also please fix the code so it is not indenting using tabs. It should use 2 spaces for indentation. This  can be changed in the bottom of sublime.